Package.swift

```diff
@@ -1,30 +1,27 @@
-// swift-tools-version:5.5
+// swift-tools-version: 5.7
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
-    name: "Segment-MoEngage",
+    name: "MoEngage-Swift-Segment",
     platforms: [
-        .iOS(.v10)
+        .iOS(.v13)
```

Comment:
@deepa-iOS-SDK  why are we supporting from iOS 13.. is it limitation from other dependencies.. if yes let mention in tech doc too

Reply:
Yes, analytics-swift by segment has minimum platform requirement of iOS 13